### DISCUSSION TITLE
Inline Tags for Insiders Only?

### DISCUSSION THREAD

**Discussion by drahenshaw** [squidfunk/mkdocs-material#6724] [Q&A] [answered]
Hi there, 

Are inline tags for insiders only? I tried to implement them using:
```
<!-- material/tags -->
```
or
```
<!-- material/my_tag_identifier -->
```
but neither way was working, nothing renders. 

I can take an entire page with the tag I want, but I'd like to just tag one section of a page with this tag instead.

Thank you!

**Comment by kamilkrzyskow** [comment]
Hello @drahenshaw,
the inline `<!-- material/tags -->` syntax is used for creating listings of the tags. The community version creates an index listing of all tagged pages, without any filtering options:
https://github.com/squidfunk/mkdocs-material/blob/5417afc00414c33f1ba5779481514a5d2651c19e/src/plugins/tags/plugin.py#L139-L143
The Insiders version allows to filter the tags while creating the index, explained in the [documentation](https://squidfunk.github.io/mkdocs-material/plugins/tags/#listing-configuration) and with [examples](https://github.com/mkdocs-material/examples/pull/17). However, I don't think it allows to only tag a section instead of a page 🤔 unless I'm not understanding your request. You can read more about the changes in the new tags plugin [here](https://github.com/squidfunk/mkdocs-material/issues/6517).

**Comment by drahenshaw** [comment]
Hi @kamilkrzyskow, 
Thanks for the response! I was looking at another [example](https://github.com/squidfunk/mkdocs-material/blob/5417afc00414c33f1ba5779481514a5d2651c19e/docs/plugins/tags.md?plain=1#L112) where an inline tag is shown as part of the MkDocs-Material documentation site, and was trying to implement it like shown here, at the top of each section. I was able to get the tag rendered but only at the top of the page.

**Reply by kamilkrzyskow** [reply, marked as answer]
Oh, those aren't tags. They're [convention](https://squidfunk.github.io/mkdocs-material/conventions/#conventions) shortcodes processed by the [`shortcodes.py` hook](https://github.com/squidfunk/mkdocs-material/blob/master/src/overrides/hooks/shortcodes.py). You'd have to replicate it yourself.
I written a bit more in another answer: https://github.com/squidfunk/mkdocs-material/discussions/6544#discussioncomment-7901513

**Reply by drahenshaw** [reply]
I see, looks like adding a replication in the the `markdown.yml` would work as long as it can source the python file?

**Reply by kamilkrzyskow** [reply]
Sure, you can copy paste it into your project and enable it in the `mkdocs.yml` file like any other hook, but as I said in the other answer you need additional custom styling you can access the currently generated custom CSS [here](https://squidfunk.github.io/mkdocs-material/assets/stylesheets/custom.14cc6f30.min.css).
